vendorsetup.sh

```
@@ -1,10 +1,83 @@
-#
-# Copyright (C) 2025 The Android Open Source Project
-# Copyright (C) 2025 SebaUbuntu's TWRP device tree generator
-#
-# SPDX-License-Identifier: Apache-2.0
-#
+FDEVICE="ares"
+DEVICE_PATH="device/xiaomi/ares"
+
+fox_get_target_device() {
+local chkdev=$(echo "$BASH_SOURCE" | grep $FDEVICE)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)